SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -526,7 +526,9 @@ CSourcePieceWise_TurbSA::CSourcePieceWise_TurbSA(unsigned short val_nDim, unsign
 CSourcePieceWise_TurbSA::~CSourcePieceWise_TurbSA(void) { }
 
 void CSourcePieceWise_TurbSA::ComputeResidual(su2double *val_residual, su2double **val_Jacobian_i, su2double **val_Jacobian_j, CConfig *config) {
-  
+	
+  transition = (config->GetKind_Trans_Model() == BC);
```

Comment:
Just move this to the routine CSourcePieceWise_TurbSA::CSourcePieceWise_TurbSA(...) and then its fine.